platforms/database.go

```diff
@@ -0,0 +1,93 @@
+package platforms
+
+import (
+	"runtime"
+	"strings"
+)
+
+// validComboString defines valid GOOS/GOARCH combinations.
+// Should be updated from https://golang.org/doc/install/source#environment.
+// This is why we leave it as a string.
+const (
+	validComboString = `
+$GOOS	$GOARCH
+android	arm
+darwin	386
+darwin	amd64
+darwin	arm
+darwin	arm64
+dragonfly	amd64
+freebsd	386
+freebsd	amd64
+freebsd	arm
+linux	386
+linux	amd64
+linux	arm
+linux	arm64
+linux	ppc64
+linux	ppc64le
+linux	mips
+linux	mipsle
+linux	mips64
+linux	mips64le
```

Comment:
I am not sure if we'll even enforce this list. These are effectively the officially supported Go platforms, which is a subset of the platforms that might run containers. I'm not sure that this is necessary to maintain in this package.